pkg/scalers/azure_eventhub_scaler.go

```diff
@@ -155,17 +155,6 @@ func parseCommonAzureEventHubMetadata(config *scalersconfig.ScalerConfig, meta *
 		meta.eventHubInfo.BlobContainer = val
 	}
 
-	meta.eventHubInfo.EventHubResourceURL = azure.DefaultEventhubResourceURL
-	if val, ok := config.TriggerMetadata["cloud"]; ok {
-		if strings.EqualFold(val, azure.PrivateCloud) {
-			if resourceURL, ok := config.TriggerMetadata["eventHubResourceURL"]; ok {
-				meta.eventHubInfo.EventHubResourceURL = resourceURL
-			} else {
-				return fmt.Errorf("eventHubResourceURL must be provided for %s cloud type", azure.PrivateCloud)
-			}
-		}
-	}
```

Comment:
I haven't been able to find where is this used in the new SDK. From my understanding, this was used for requesting the tokens (we requested them using http client) and now the SDK manages it in a better way